include/utility/json_utility.hpp

```diff
@@ -926,5 +926,68 @@ inline bool isFruPowerOffOnly(const nlohmann::json& i_sysCfgJsonObj,
             (i_sysCfgJsonObj["frus"][i_vpdFruPath].at(0)["powerOffOnly"]));
 }
 
+/**
+ * @brief API which tells if the FRU 's VPD is recollectable at runtime
```

Comment:
FRU 's -> FRU's.

Reply:
Also, IMO instead of using "recollectable", should we use "collectable" here and at all other places?
As this basically re-presents if the FRU is collectable at runtime or not? 
We can give this a thought.